Services/Interfaces/web_interface/templates/index.html

```diff
@@ -2,6 +2,9 @@
 {% set active_page = "home" %}
 {% block body %}
     <div update-url="{{ url_for('api.announcements') }}" class="alert alert-danger text-center d-none" role="alert" id="annoncementsAlert"></div>
+    <div class="d-none alert alert-success text-center mt-2" role="alert">
+        <h5><i class="far fa-bell"></i> Good news ! OctoBot version <span update-url="{{ url_for('api.upgrade_version') }}" id="upgradeVersion"></span> is available. <button route="{{ url_for('commands', cmd='update') }}" type="button" class="btn btn-warning waves-effect">Upgrade now <i class="fas fa-cloud-download-alt"></i></button></h5>
```

Comment:
I'm wondering if we should make it take less space in mobile version 
![image](https://user-images.githubusercontent.com/9078616/113521859-80024000-959c-11eb-8627-6bec28530da7.png)

Reply:
if we could fit it all in one line on mobile, it would be really nice

Reply:
Right :+1:

Reply:
Done 
![image](https://user-images.githubusercontent.com/10454670/113744816-557fc680-9705-11eb-8408-35c9a349231f.png)

Reply:
Updated to one line :

![image](https://user-images.githubusercontent.com/10454670/113747831-156e1300-9708-11eb-90f0-9b3ae3646ae6.png)